### PR TITLE
added models class

### PR DESCRIPTION
**Pull request by abimanu-devl**

### CODE REVIEW

File: models/Enquiry.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    reason: {
+        type: String,
+        required: true,
+        trim: true
+    },
+    date: {
+        type: String,
+        required: true
+    },
+    users: [{ type: mongoose.Schema.Types.ObjectId, required: false, ref: 'Users' }]
```
Comment (himashatw) [inline models/Enquiry.js:18]: should be required: true

Comment (himashatw) [inline models/Enquiry.js:18]: required: true

File: models/Payment.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    },
+    expireDate: {
+        type: String,
+        required: true
+    },
+    cvv: {
+        type: String,
+        required: true
+    },
+    users: [{ type: mongoose.Schema.Types.ObjectId, required: false, ref: 'Users' }]
```
Comment (himashatw) [inline models/Payment.js:22]: required: true

**Review by himashatw — CHANGES_REQUESTED** [review]
do the changes

**Comment by abimanu-devl** [comment]
okey, I'll.

**Comment by abimanu-devl** [comment]
done.

**Pull request closed without merging**